pages/linux/mopac.md

```diff
@@ -0,0 +1,12 @@
+# mopac
+
+> MOPAC (Molecular Orbital PACkage) is a semiempirical quantum chemistry program based on Dewar and Thiel's NDDO approximation.
+> More information: <https://github.com/openmopac/mopac>.
+
+- Perform calculations according to an input file (`.mop`, `.dat`, and `.arc`):
+
+`mopac {{path/to/input_file}}`
+
+- Minimal working example with HF that writes to the current directory and streams the output file:
+
+`touch test.out; echo "PM7\n#comment\n\nH 0.95506 0.05781 -0.03133\nF 1.89426 0.05781 -0.03133" > test.mop; mopac test.mop & tail -f test.out`
```

Comment:
the docs mention that the naming can get messed up when the endings i mentioned are _not_ used. but otherwise yes, the output should always be written to {{filename}}.out